public/js/Tree.js

```diff
@@ -929,9 +929,17 @@ Tree.prototype.centerOnNode = function(node)
 	{
 		var style = self.locals.style.get(this);
 
+		isResource = this.__data__.id.charAt(0) === 'r';
```

Comment:
We should modify this code to follow the style guide. Variables are snake case, and scope has { on newline.